game.rb

```diff
@@ -0,0 +1,38 @@
+class Game
+
+  require_relative 'score'
+
+  attr_reader :frames
+
+  def initialize
+    @frames = Array.new(10){ [] }

```

Comment:
Interesting - I've never seen this kind of array syntax! Given that it's dealing with frames I'm assuming this is your way of setting the maximum amount of elements in the array as 10. But what I don't get is what ```{ [] }``` that is for?